### ISSUE TITLE
TSV Parses ""'s in Columns Incorrectly

### ISSUE THREAD

**Issue by agrow** [d3/d3-dsv#21]
Greetings!

TLDR: Sections that begin with a quoted item but includes other text afterwards, such as:
`"Hello" world`
Will parse as 
`"Hello"<tab> world`
which is two entries rather than one.

My data includes plain text and is exported to the tsv file correctly (verified with visual white space viewed in Word). However, when it is imported via d3.tsv, it splits an entry such as the one above into two, shoving over all my other data.
I do not have time to make an isolated test right now, but here are some screenshots of the incorrect parse (and one photo including an adjacent correct parse).

One instance:
![2016-12-13_1404](https://cloud.githubusercontent.com/assets/1690642/21161700/8cf512e2-c13e-11e6-8173-39a34e3e42a8.png)
![2016-12-13_1403](https://cloud.githubusercontent.com/assets/1690642/21161698/8cf21b1e-c13e-11e6-8f9c-3bf6b82318d6.png)
![2016-12-13_1405](https://cloud.githubusercontent.com/assets/1690642/21161699/8cf26e98-c13e-11e6-9ebc-e8cb08172150.png)

Another instance:
![2016-12-13_1418](https://cloud.githubusercontent.com/assets/1690642/21161824/1b8079c0-c13f-11e6-814f-4f40e780c852.png)
![2016-12-13_1411](https://cloud.githubusercontent.com/assets/1690642/21161696/8ce2fe68-c13e-11e6-8d0f-f5a57531b450.png)
![2016-12-13_1413](https://cloud.githubusercontent.com/assets/1690642/21161695/8cdfb064-c13e-11e6-93e5-26f61db7e4d7.png)
Includes correct parse ^

**Comment by mbostock** [comment]
Per [RFC 4180](https://tools.ietf.org/html/rfc4180):

> If fields are not enclosed with double quotes, then double quotes may not appear inside the fields.

If you want the parsed value of the field to be `"Hello" world`, the serialized text for that field should be `"""Hello"" world"`.

**Comment by agrow** [comment]
That link is for CSV, not TSV.

Also, `Hello "world"` parses correctly. Double quotes "may not appear" only when they begin the field. In a TSV.

**Comment by mbostock** [comment]
This repo generalizes CSV as defined in RFC 4180 to support other delimiters besides comma, including tab (`\t`). But otherwise it adheres to that specification.

The fact that `Hello "world"` parses as expected does not mean that the input is correctly formatted; the correct input format should be `"Hello """world"""` in that case. This library doesn’t validate the input, and so its behavior is undefined if you give it invalid input.
